kornia/morphology/morphology.py

```diff
@@ -563,3 +563,55 @@ def bottom_hat(
         )
         - tensor
     )
+
+
+def skeletonize(tensor: torch.Tensor, kernel: Optional[torch.Tensor] = None) -> torch.Tensor:
```

Comment:
```suggestion
def skeletonize(tensor: Tensor, kernel: Tensor | None = None) -> Tensor:
```
`from __future__ import annotations`
`from kornia.core import Tensor`